app/misc.py

```diff
@@ -229,6 +232,15 @@ def get_global_stylesheet(self):
         return ''
 
 
+def is_target_user_admin(uid):
+    try:
+        check_admin = UserMetadata.get((UserMetadata.uid == uid) and (UserMetadata.key == 'admin') and (UserMetadata.value == '1'))
```

Comment:
Didn't catch this one in the last review, but this won't work. When using peewee you have to use `&` instead of `and`